src/adapters/sqlite/type.js

```diff
@@ -13,12 +13,19 @@ export type SQL = string
 export type SQLiteArg = string | boolean | number | null
 export type SQLiteQuery = [SQL, SQLiteArg[]]
 
+export type MigrationEvents = {
+  onSuccess?: () => void,
+  onStarted?: () => void,
+  onFailure?: (error: string) => void,
```

Comment:
I think for simplification (no `?.()`), those can be required. So, if you're passing migrationEvents at all, you must specify all three.

Reply:
corrected.